tests/testthat/test-arg_is_.R

```diff
@@ -136,14 +137,6 @@ test_that("function", {
   expect_silent(arg_is_function(g, f = NULL, allow_null = TRUE))
 })
 
-test_that("sorted", {
-  expect_silent(arg_is_sorted(a = 1:5, b = 6:10))
-  expect_error(arg_is_sorted(a = 5:1, b = 6:10))
-  expect_error(arg_is_sorted(b = NULL))
-  expect_silent(arg_is_sorted(b = NULL, allow_null = TRUE))
-})
-
-
```

Comment:
These weren't used anywhere and checkmate supports a sorted check for some data types, so I figured we could rely on that when we need it.